.github/workflows/fevm-contract-tests.yaml

```diff
@@ -71,8 +71,7 @@ jobs:
         id: docker-build
         working-directory: ipc/fendermint
         run: |
-          export PATH="$PATH:/home/runner/.config/.foundry/bin"
-          docker pull ghcr.io/consensus-shipyard/fendermint:latest
+          export PATH="$PATH:/home/runner/.config/.foundry/bin" && make docker-build
```

Comment:
Isn't this gonna crazy expensive? IIRC we build a Docker image in another job, can't we reuse that Docker image?